src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -16,7 +18,9 @@ public class JwtTokenReader : IJwtTokenReader
         private const string MicrosoftDiscoveryEndpoint =
             "https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration";
 
-        private readonly string _applicationId;
+        private const string AudienceKeyName = JwtClaimTypes.Audience;
```

Comment:
Minor remark, but this private member can be removed if you use JwtClaimTypes.Audience directly instead of the `AudienceKeyName` member

Reply:
done